fetch/stale-while-revalidate/resources/frame-removal.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test that frame removal cancels its revalidations</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
+<body>
+<script>
+var request_token = token();
+
+window.onmessage = async function(e) {
+  if (e.data == 'start') {
+    const response = await fetch(`stale-script.py?token=` + request_token);
+    // Wait until resource is completely fetched to allow caching before next fetch.
+    const body = await response.text();
+    const response2 = await fetch(`stale-script.py?token=` + request_token);
```

Comment:
How about renaming response2 to responseFromStale or something like that to make it easier to understand.